br/pkg/restore/import.go

```diff
@@ -113,23 +113,29 @@ type ImporterClient interface {
 		storeID uint64,
 	) (import_sstpb.ImportSSTClient, error)
 
+	CloseGrpcClient() error
+
 	SupportMultiIngest(ctx context.Context, stores []uint64) (bool, error)
 }
 
 type importClient struct {
-	mu         sync.Mutex
 	metaClient split.SplitClient
-	clients    map[uint64]import_sstpb.ImportSSTClient
-	tlsConf    *tls.Config
+	mu         sync.Mutex
+	// used for any request except the ingest reqeust
+	conns map[uint64]*grpc.ClientConn
+	// used for ingest request
+	ingestConns map[uint64]*grpc.ClientConn
```

Comment:
Would you explain why ingest requests require isolated connections here?

Reply:
It seems that the grpc connection isn't closed when br task exits.
In the BRIE via SQL, I'm worried that it would be leak.